### ISSUE TITLE
XMIRs lacking XSD schema validation in YAML packs' `document` key

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#500]
Consider this pack:

```yaml
sheets:
  - /org/eolang/lints/critical/void-attributes-not-higher-than-other.xsl
asserts:
  - /defects[count(defect[@context and @severity='critical'])=1]
document: |
  <object>
    <o name="foo">
      <o base="∅" name="x"/>
      <o base="Q.foo" name="z"/>
      <o base="∅" name="y"/>
    </o>
  </object>
```

If the `document` key is present, then we should validate its content against XSD [schema](https://www.eolang.org/XMIR.xsd). Let's make this check a separate unit test in `LtByXslTest`

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel make sense, let's do it

**Comment by 0crat** [comment]
@h1alexbel Hey there, bug detective! 🕵️‍♂️ You've just scored 8 points for spotting that sneaky bug - nice work! Your eagle eye is making our project better by the day. Keep 'em coming if you see anything else that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) quite right. Remember, [bugs are welcome](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) here, so [report away](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html)! Your current score is sitting pretty at [+133](https://www.eolang.org/zerocracy/objectionary-vitals.html). Happy hunting!
